packages/graphql-tool-utilities/augmentations.ts

```diff
@@ -0,0 +1,16 @@
+import {GraphQLProjectConfig} from 'graphql-config/lib/GraphQLProjectConfig';
+
+declare module 'graphql-config/lib/GraphQLProjectConfig' {
+  interface GraphQLProjectConfig {
+    resolveProjectName(defaultName?: string): string;
+  }
+}
+
+function resolveProjectName(
+  this: GraphQLProjectConfig,
+  defaultName = 'GraphQL',
```

Comment:
This is our new default project name. I am happy to change this to just about anything if anyone has opinions. 😄